rllib/env/env_runner_group.py

```diff
@@ -1202,7 +1201,19 @@ def _make_worker(
             tune_trial_id=self._tune_trial_id,
         )
 
-        return worker
+        if worker_index == 0:
+            return self.env_runner_cls(**kwargs)
+
+        pg_bundle_idx = (
```

Comment:
Does this assignment per PG order always work?

Reply:
Let's hope so :D
Yeah, it's indexing into the first n bundles, not counting the 0th bundle for the Trial (Algorithm main process/actor). These are always EnvRunner reserved. The second m bundles - if applicable - are then eval EnvRunners.